### PR TITLE
Feat: Allow usage without remote module

### PR DESCRIPTION
**Pull request by timfish**
cc @koenoe

This allows usage of `sentry-electron` with the `remote` module disabled in renderers:
```
const mainWindow = new BrowserWindow({
  webPreferences: {
    enableRemoteModule: false
  }
})
```
To support this you have to pass the app name to sentry options:
```
const { init } = require('@sentry/electron');

init({
  dsn: 'https://343434343434343@sentry.io/5656565',
  appName: 'My Great App',
});
```

### TODO
Add some tests!

### CODE REVIEW

**Comment by timfish** [comment]
@HazAT the build is failing for the same reason as your `feat/allow-no-remote` branch. `Dsn` has moved from `utils` to `core` but isn't exported?

File: src/main/backend.ts [hunk 1]
```diff
@@ -4,16 +4,17 @@ import {
   captureEvent,
   captureMessage,
   configureScope,
+  Dsn,
```
Comment (HazAT) [inline src/main/backend.ts]:
```suggestion
```
`Dsn` lives in `@sentry/utils` now.

File: src/main/uploader.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
+import { Dsn } from '@sentry/core';
```
Comment (HazAT) [inline src/main/uploader.ts]:
```suggestion
```
Same here

File: src/main/uploader.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
 import { Event } from '@sentry/types';
-import { Dsn, logger } from '@sentry/utils';
+import { logger } from '@sentry/utils';
```
Comment (HazAT) [inline src/main/uploader.ts]:
```suggestion
import { Dsn, logger } from '@sentry/utils';
```

File: src/main/backend.ts [hunk 4]
```diff
@@ -4,16 +4,17 @@ import {
...
   getCurrentHub,
   Scope,
 } from '@sentry/core';
 import { NodeBackend } from '@sentry/node/dist/backend';
 import { Event, EventHint, Severity, Transport, TransportOptions } from '@sentry/types';
-import { Dsn, forget, logger, SentryError } from '@sentry/utils';
+import { forget, logger, SentryError } from '@sentry/utils';
```
Comment (HazAT) [inline src/main/backend.ts]:
```suggestion
import { Dsn, forget, logger, SentryError } from '@sentry/utils';
```

**Pull request merged**